src/backend.f90

```diff
@@ -178,6 +193,21 @@ subroutine field_max_mean(self, max_val, mean_val, f, enforced_data_loc)
     end subroutine field_max_mean
   end interface
 
+  abstract interface
+    subroutine field_set_face(self, f, c_start, c_end, face)
+      !! Set a face in field
```

Comment:
I can't figure out from the commen what this should be doing

Reply:
Updated this bit with a detailed description